scripts/archived_scripts/community_scripts/amm_price_example.py

```diff
@@ -6,7 +6,7 @@
 
 class AmmPriceExample(ScriptStrategyBase):
     """
-    This example shows how to call the /amm/price Gateway endpoint to fetch price for a swap
+    This example shows how to call the /uniswap/price Gateway endpoint to fetch price for a swap
```

Comment:
In this comment it should be amm (since it calling amm/price endpoint , no need to change to uniswap